src/map/map.cpp

```diff
@@ -3784,8 +3784,10 @@ int map_readallmaps (void)
 		struct map_data *mapdata = &map[i];
 		char map_cache_decode_buffer[MAX_MAP_SIZE];
 
+#ifdef DEBUG
```

Comment:
Above the loop and after the loop the same messages as everywhere should be added.
Maybe harmonize the wording as well!
Cant suggest that on Github Web, sorry!